tools/autobuild/build-esp8266-latest.sh

```diff
@@ -1,61 +0,0 @@
-#!/bin/bash
-
-PYTHON3=python3
-yaota8266=$HOME/yaota8266
-
-# for debugging
-#exec &> /tmp/esp-log-$$.txt
-
-# function for building firmware
-function do_build() {
-    descr=$1
-    board=$2
-    shift
-    shift
-    echo "building $descr $board"
-    build_dir=/tmp/esp8266-build-$board
-    $MICROPY_AUTOBUILD_MAKE $@ BOARD=$board BUILD=$build_dir || exit 1
-    mv $build_dir/firmware-combined.bin $dest_dir/$descr$fw_tag.bin
```

Comment:
The main firmware is `firmware-combined.bin` and I don't think the generic build function handles this filename.  Probably FWBIN (in esp8266 Makefile) should be changed to just `FWBIN = firmware.bin`.

Reply:
Done.